infrastructure/api-configuration/api-server-instance-user-data.tpl.sh

```diff
@@ -22,8 +22,10 @@ cp nginx.conf /etc/nginx/nginx.conf
 service nginx restart
 
 # install and run docker
-apt-get install docker-ce -y
-service docker restart
```

Comment:
I assume it does start automatically after the installation process is over?

Reply:
yeah it usually does this is just in case there is a reconcilable error